src/utils/balance.utils.ts

```diff
@@ -101,3 +102,7 @@ export function calculateValuePerChain(balances: IUserBalance[]): ChainValue[] {
     }
     return result
 }
+
+export function printableUsdc(baseUnitsAmount: bigint): string {
+    return Number(formatUnits(baseUnitsAmount, 6)).toFixed(2)
+}
```

Comment:
Thought: seems a bit hardcoded

Reply:
I think is ok for now, PW is only usdc and usdc is 6 decimals